File: app/src/main/java/com/enipro/presentation/generic/industry/IndustryActivity.java
package com.enipro.presentation.generic.industry;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

import com.enipro.R;
import com.enipro.model.Utility;

import butterknife.BindView;
import butterknife.ButterKnife;

public class IndustryActivity extends AppCompatActivity {

    @BindView(R.id.industry_list_recycler)
    RecyclerView mRecyclerView;

    private IndustryAdapter industryAdapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_industry);
        ButterKnife.bind(this);

        mRecyclerView.setHasFixedSize(true);
        mRecyclerView.addItemDecoration(new Utility.DividerItemDecoration(this));

        LinearLayoutManager linearLayoutManager = new LinearLayoutManager(this);
        linearLayoutManager.setOrientation(LinearLayoutManager.VERTICAL);
        mRecyclerView.setLayoutManager(linearLayoutManager);


        // Create an adapter and pass items into adapter
        industryAdapter = new IndustryAdapter(this, null);
        mRecyclerView.setAdapter(industryAdapter);

    }
}

File: app/src/main/java/com/enipro/presentation/generic/industry/IndustryActivity.kt
package com.enipro.presentation.generic.industry

import android.os.Bundle
import android.os.PersistableBundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import com.enipro.R
import com.enipro.model.Utility
import kotlinx.android.synthetic.main.activity_industry.*

class IndustryActivity : AppCompatActivity() {

    private var adapter: IndustryAdapter? = null


    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
        setContentView(R.layout.activity_industry)

        industry_list_recycler.setHasFixedSize(true)
        industry_list_recycler.addItemDecoration(Utility.DividerItemDecoration(this))

        val linearLayoutManager = LinearLayoutManager(this)
        linearLayoutManager.orientation = LinearLayoutManager.VERTICAL
        industry_list_recycler.layoutManager = linearLayoutManager

        adapter = IndustryAdapter(this, resources.getStringArray(R.array.industries).toList())
        industry_list_recycler.adapter = adapter
    }
}